D/DynamicPolynomials/Compat.toml

```diff
@@ -25,3 +25,8 @@ MultivariatePolynomials = "0.2"
 ["0.1.2-0.1.4"]
 MultivariatePolynomials = "0.2.1-0.2.8"
 Reexport = "0.2"
+
+["0.2"]
+MultivariatePolynomials = ["0.2.1-0.2"]
```

Comment:
https://github.com/JuliaRegistries/General/pull/111#discussion_r277135089

Reply:
I just retriggered Registrator but it didn't change the line. What should I do ?